### ISSUE TITLE
So detectable

### ISSUE THREAD

**Issue by Izuyazep** [Ayhuuu/Creal-Stealer#6]
The .exe is so dectectable even if obfuscated
![image](https://user-images.githubusercontent.com/96847178/225082745-416641d8-39fb-4aad-817d-2c20140b89f0.png)

**Comment by exit7** [comment]
> The .exe is so dectectable even if obfuscated ![image](https://user-images.githubusercontent.com/96847178/225082745-416641d8-39fb-4aad-817d-2c20140b89f0.png)

get a better crypter than lmao you're just using free crypters that it becomes detectable

**Comment by mzjwijznzmz** [comment]
Can u encrypt my file Ill pay u

**Comment by mzjwijznzmz** [comment]
> > The .exe is so dectectable even if obfuscated ![image](https://user-images.githubusercontent.com/96847178/225082745-416641d8-39fb-4aad-817d-2c20140b89f0.png)
> 
> get a better crypter than lmao you're just using free crypters that it becomes detectable

Can u encrypt my file ill pay u

**Comment by BleakLeaks** [comment]
> > > The .exe is so dectectable even if obfuscated ![image](https://user-images.githubusercontent.com/96847178/225082745-416641d8-39fb-4aad-817d-2c20140b89f0.png)
> > 
> > 
> > get a better crypter than lmao you're just using free crypters that it becomes detectable
> 
> Can u encrypt my file ill pay u

Yes I can, please message me on telegram at https://t.me/CvCZeus or on discord or on discord !Lambo#2911

**Comment by exit7** [comment]
> Can u encrypt my file Ill pay u

no im not crypting something for a skid lmao

**Comment by mzjwijznzmz** [comment]
> > Can u encrypt my file Ill pay u
> 
> no im not crypting something for a skid lmao

ur a retard btw, not just for this scenario for future transactions in ur life, mf passing free money. Unless of course u cant do shit and ur the skid pretending to know everything

**Comment by exit7** [comment]
> > > Can u encrypt my file Ill pay u
> > 
> > 
> > no im not crypting something for a skid lmao
> 
> ur a retard btw, not just for this scenario for future transactions in ur life, mf passing free money. Unless of course u cant do shit and ur the skid pretending to know everything

you're using a builder skidded off of empyrean and w4sp lmao fuck off

**Comment by mzjwijznzmz** [comment]
> > > > Can u encrypt my file Ill pay u
> > > 
> > > 
> > > no im not crypting something for a skid lmao
> > 
> > 
> > ur a retard btw, not just for this scenario for future transactions in ur life, mf passing free money. Unless of course u cant do shit and ur the skid pretending to know everything
> 
> you're using a builder skidded off of empyrean and w4sp lmao fuck off

And I grabbed about 200 people with it nigga whats ur point LMFAOO

**Comment by exit7** [comment]
grabbing 200 people with a non fud grabber is not even normal unless you targeted some random dumbass community to get your hits

**Comment by exit7** [comment]
and usually dumbasses dont have money

**Comment by mzjwijznzmz** [comment]
> and usually dumbasses dont have money

My shit is already detected by 6 avs only, if it was 0 avs I would have grabbed about a thousand not even capping. and yes you would be surprised how many retards are using discord. Idk who u talking about but I aint a broke nigga I pay good money for the right service

**Comment by mzjwijznzmz** [comment]
Some guy encrypted it for me but it got detected by time so I need someone to re encrypt it

**Comment by exit7** [comment]
> > and usually dumbasses dont have money
> 
> My shit is already detected by 6 avs only, if it was 0 avs I would have grabbed about a thousand not even capping. and yes you would be surprised how many retards are using discord. Idk who u talking about but I aint a broke nigga I pay good money for the right service

put ur payload in a zip file and encrypt it with a password gg fud (until they put the exe and not the zip into virustotal after decrypting it) thats the freest way you can half fud it

**Comment by exit7** [comment]
other ways are using cassandra or find sum free crypter on github

**Comment by mzjwijznzmz** [comment]
> > > and usually dumbasses dont have money
> > 
> > 
> > My shit is already detected by 6 avs only, if it was 0 avs I would have grabbed about a thousand not even capping. and yes you would be surprised how many retards are using discord. Idk who u talking about but I aint a broke nigga I pay good money for the right service
> 
> put ur payload in a zip file and encrypt it with a password gg fud (until they put the exe and not the zip into virustotal after decrypting it) thats the freest way you can half fud it

Yeah Im already using this method got a pass on it, the problem is when they run the exe itself

**Comment by Linxo-glitch** [comment]
yo im new to all this shit and ive tried using empyrean but idk if its a malware or not but if its not how would i even get people to download and open it ?
